src/NotFound.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  findByRole, findByText,
```

Comment:
I suggest using `screen` instead of importing and using individual queries.
Reasoning and examples here: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

Reply:
Updated